### ISSUE TITLE
Segmentation fault after choosing field with space

### ISSUE THREAD

**Issue by utkonos** [lastpass/lastpass-cli#177]
When trying to access the Secure Note type "Software License" and get to the field "License Key", if I use this command it works:

    $ lpass show --field='License Key' 1234567890

This command causes the following error:

    $ lpass show --field=License Key 1234567890
    Segmentation fault: 11

**Comment by bcopeland** [comment]
This looks like the bug fixed in d9c5b297fa03c45422520ef160fe410a74388f18 -- can you tell me which version you have?

**Comment by bcopeland** [comment]
Ping, can you verify this is an issue on master?

**Comment by nicr9** [comment]
I have verified that this was fixed by d9c5b29.

Here are the steps that I took to repro the error and the fix in `master` (in the name of brevity, stdout of `make` has been hidden):

```bash
$ git checkout 82c6594
Previous HEAD position was d9c5b29... show: fix crash when requested field is not found
HEAD is now at 82c6594... notes: use note-type not note_type

$ make; sudo make install
...
removed ‘/usr/bin/lpass’
‘lpass’ -> ‘/usr/bin/lpass’
removed ‘/usr/share/bash-completion/completions/lpass’
‘contrib/lpass_bash_completion’ -> ‘/usr/share/bash-completion/completions/lpass’

$ lpass show --field=License Key notes/test_177
Segmentation fault (core dumped)

$ git checkout master
Previous HEAD position was 82c6594... notes: use note-type not note_type
Switched to branch 'master'
Your branch is up-to-date with 'origin/master'.

$ make; sudo make install                                                                                                                                             ...
removed ‘/usr/bin/lpass’
‘lpass’ -> ‘/usr/bin/lpass’
removed ‘/usr/share/bash-completion/completions/lpass’
‘contrib/lpass_bash_completion’ -> ‘/usr/share/bash-completion/completions/lpass’

$ lpass show --field=License Key notes/test_177
Error: Could not find specified field 'License'.
```

I believe this issue can be closed.
